backend/graph/service.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express')
```

Comment:
if this (the whole nodejs setup) is only for your local testing then probably doesn't belong into the repository,
it will not anyway be used on the server side

Reply:
It was important for me because I wanted to test CORS access works.
I don't know if other people want to test it locally.
I can remove it anyway.
Then there is no meaning to put files in a `test` directory.

Reply:
I have removed Node dependencies for a local test and the `test` directory.